### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joelmcg**
Added notes about high-gluten flour and an optional ice bath

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ You want a STRONG mixer for this recipe. Doing it by hand would be impossible an
 
 1. Food Grade Lye– I use this stuff https://www.amazon.com/Sodium-Hydroxide-Micro-Beads-Bottle/dp/B0838Q24Q5/ref=sr_1_5?keywords=food+grade+lye&qid=1639332427&sr=8-5
 2. Diastatic malt powder or malt syrup– lots of choices here that will affect the character of your bagels. I like using dark malt syrup for brewing (e.g., https://www.amazon.com/gp/product/B003BCZXM4/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) but malt powder is an excellent way to go too and more in line with commercial NYC bagels
-3. Vital wheat gluten– you need your flour to be super high protein for this recipe to work properly. We’re gonna spike AP to make sure we get there. E.g., https://www.bobsredmill.com/vital-wheat-gluten.html
+3. Vital wheat gluten– you need your flour to be super high protein for this recipe to work properly. We’re gonna spike AP to make sure we get there. E.g., https://www.bobsredmill.com/vital-wheat-gluten.html Alternatively, you can use a high-protein flour like King Arthur Sir Lancelot or Gold Medal All Trumps in place of the AP flour/gluten mixture.
```
Comment (andrewkern) [inline README.md:13]: so using Sir Lancelot would, according to my calculations, yield significantly less protein than what i'm recommending here.

**Review by andrewkern — CHANGES_REQUESTED** [review]
mind cutting the Sir Lancelot sentence? I think that's not enough gluten!!

**Comment by andrewkern** [comment]
Actually I take this back- it should be very close in protein content

File: README.md (hunk 1 above)
Comment (joelmcg) [reply README.md:13]: Sir Lancelot has 14.2% protein. King Arthur All-Purpose flour has 11.7%, VWG 75%.

Using 1 c = 128 g:

992 g SL * 0.142 = 141 g protein
949 g AP * 0.117 + 43 g VWG * 0.75 = 143 g protein

Comment (joelmcg) [reply README.md:13]: I can still take it out though

**Pull request merged**